xdesign/acquisition.py

```diff
@@ -278,12 +278,13 @@ def measure(self, phantom, sigma=0):
 
         Parameters
         ----------
-        sigma : float >= 0
-            The standard deviation of the normally distributed noise.
+        perc : float >= 0
+            Percentage of Gaussian noise.
         """
         newdata = self._measure_helper(phantom)
+        std = perc * newdata
         if sigma > 0:
-            newdata += newdata * np.random.normal(scale=sigma)
+            newdata += newdata * np.random.normal(scale=std)
```

Comment:
```python
# the proposed noise model
newdata = newdata + newdata * np.random.normal(scale=perc * newdata)
# this is most probably equivalent to the next line
newdata = 2 * newdata
```

The above model is definitely wrong. I think a correct noise model would be one of the two below:
```python
# the noise has a standard normal distribution around perc percent 
newdata = newdata + newdata * perc * np.random.normal(scale=1)
# the 95% of the noise is within perc percent
newdata = newdata + newdata * perc / 2 * np.random.normal(scale=1)
```

Reply:
Right, what about?
```python
newdata += np.random.normal(scale=newdata * perc)
```
I think this is the same as the first one and more intuitive?

Reply:
I want to know why this option:
```python
newdata += np.random.normal(scale=newdata * perc) #option A
```
is more correct than this option:
```python
newdata += newdata * perc * np.random.normal(scale=1) #option B
```
These options do not give the same result.

Reply:
Option B please.